src/js/index.js

```diff
@@ -1,5 +1,167 @@
-(function () {
+/*global $ */
+'use strict';
 
-  console.log('WOLOX');
+//Manejo de animaciones cuando abre el sitio
+function openBars(time){
+	$('.copyright').animate({
+		bottom: '+=200'
+	},time,function(){});
 
-})();
+	$('header').animate({
+		marginTop: '+=400'
+	},time,function(){});
+}
+
+var slide = 1;
+
+//Manejo del slideshow
+$('.backcontrols').hover(function(){
+	if ($('.copyright').css('bottom') ==='-200px'){
+		openBars(1000);
+	}
+},function(){});
+
+
+function pasar(slide){
+	$('body').css('background-image', 'url("../assets/slide' + slide + '.jpg")');
+}
+
+
+var paso;
+function slider(){
+	paso = setInterval(function(){
+		if(slide>=4) slide=0;
```

Comment:
always use { } , leave a space after the if and between every +, =, -, >=, <=, &, *.

in this case it should look something like this:

```
if (slide >= 4) {
    slide = 0;
}
```